graylog2-web-interface/src/views/logic/widgets/MessagesWidget.ts

```diff
@@ -26,8 +27,9 @@ import type { WidgetState } from './Widget';
 import type { QueryString, TimeRange } from '../queries/Query';
 
 export default class MessagesWidget extends Widget {
-  constructor(id: string, config: any, filter: string | undefined | null, timerange: TimeRange | undefined | null, query: QueryString | undefined | null, streams: Array<string> = []) {
-    super(id, MessagesWidget.type, config, filter, timerange, query, streams);
+  // eslint-disable-next-line default-param-last
+  constructor(id: string, config: any, filter: string | undefined | null, timerange: TimeRange | undefined | null, query: QueryString | undefined | null, streams: Array<string> = [], filters?: FiltersType) {
```

Comment:
Afaics we do not rely on the default value for the `streams` parameter, so we can just remove it. This way we do not need to suppress `default-param-last` for the `filters` parameter.